build/linux/installer/conf/70-rsyslog-forward-mdsd-ci.conf

```diff
@@ -1,5 +1,10 @@
-$ModLoad omuxsock 
-$OMUxSockSocket /var/run/mdsd-ci/default_syslog.socket 
 template(name="MDSD_RSYSLOG_TraditionalForwardFormat" type="string" string="<%PRI%>%TIMESTAMP% %HOSTNAME% %syslogtag%%msg:::sp-if-no-1st-sp%%msg%")
-$OMUxSockDefaultTemplate MDSD_RSYSLOG_TraditionalForwardFormat
-*.* :omuxsock:
\ No newline at end of file
+
+# Forwarding all events through TCP port
+*.* action(type="omfwd"
+template="MDSD_RSYSLOG_TraditionalForwardFormat"
+queue.type="LinkedList"
+action.resumeRetryCount="-1"
```

Comment:
Does this mean infinite retry? will this put pressure memory if MDSD failed to ingest the syslog data?

Reply:
no, since there is also an interval for retries which grows with the number of retries. from rsyslog docs

If multiple retries fail, the interval is automatically extended. This is to prevent excessive resource use for retries. After each 10 retries, the interval is extended by itself. To be precise, the actual interval is (numRetries / 10 + 1) * action.resumeInterval. Using the default value of 30, this means that on the 10th try the suspension interval will be 60 (seconds) and after the 100th try it will be 330 (seconds).